controllers/expenses.controller.js

```diff
@@ -50,4 +50,35 @@ exports.remove = (req, res) => {
   }).catch(err => {
     res.status(422).send({answer: err});
   })
+}
+
+exports.edit = (req, res) => {
+  const {id} = req.params;
+  const {shop, spend} = req.body;
+  const errorsArr = [];
+
+  if (id) {
+    if (!shop || !spend) {
```

Comment:
So you wrote, that even if one isn't passed there got to be validation error, that's incorrect. What if user wants to update only one field